module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -39,123 +31,70 @@ CREATE TABLE customer (
 
 ```
 
-We can see that the schema definition for the `Customer` has a `name` which is a type `string` and a `zipcode` that is of type `number`.
+We can see that the schema definition for the `customer` has a `name` which is a type `string` and a `zipcode` that is of type `number`.
 
 ### The Difference between data model and schema
 
-The database schema is one that contains list of attributes and instructions to tell the database engine how data is organised whereas Data model is a collection of conceptional tools for describing data, data-relationship and consistency constraints.
+The database schema is one that contains list of attributes and instructions to tell the database engine how data is organised whereas data model is a collection of conceptional tools for describing data, data-relationship and consistency constraints.
 
 ## Introduction to CRUD operations
 
 When we are building APIs, we want our models to provide four basic types of functionality. The model must be able to Create, Read, Update, and Delete resources. Computer scientists often refer to these functions by the acronym `CRUD`. A model should have the ability to perform at most these four functions in order to be complete.
 
 ### CRUD and REST
 
+### What is REST?
+
+REST is a set of architectural constraints. API developers can implement REST in a variety of ways. When a client request is made via a RESTful API, it transfers a representation of the state of the resource to the requester or endpoint. This information, or representation, is delivered in one of several formats, JSON is the most generally popular file format to use.
+
+### CRUD (CREATE, READ, UPDATE, DELETE)
+
 In a REST environment, CRUD often corresponds to the HTTP methods `POST`, `GET`, `PUT`, and `DELETE`, respectively. These are the fundamental elements of a persistent storage system.
 
 For example, imagine we are working with a system that is keeping track of meals and their corresponding prices for a restaurant. Let’s look at how we would implement CRUD operations.
 
-#### Create
+- #### Create
```

Comment:
i think this was fine before, i dont think using a bullet point followed by a header is a common practice